### ISSUE TITLE
Cut a new release?

### ISSUE THREAD

**Issue by simoncozens** [googlefonts/glyphsLib#712]
I have people asking for Glyphs3 support but I would prefer to point them to a released glyphsLib rather than force a git-branch requirement.

**Comment by khaledhosny** [comment]
Can we merge #706, #707, #708, #709, and #711 first? All but the last affect Glyphs 3 support, and the last affects anyone using the EraseOpenCorners filter.

**Comment by simoncozens** [comment]
Agree.

**Comment by khaledhosny** [comment]
There have been 7 6.0.0 betas, time for a stable release?

**Comment by khaledhosny** [comment]
I guess so. Should I copy all the release notes since last stable release?

**Comment by anthrotype** [comment]
yeah, makes sense to consolidate all those in the stable 6.0.0 changelog

**Comment by khaledhosny** [comment]
[Don](https://github.com/googlefonts/glyphsLib/releases/tag/v6.0.0)
